src/containers/About/About.js

```diff
@@ -0,0 +1,182 @@
+/*
+Copyright 2020 The Tekton Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { injectIntl } from 'react-intl';
+import { Table } from '@tektoncd/dashboard-components';
+import { InlineNotification } from 'carbon-components-react';
+import { getInstallProperties } from '../../api';
+import { isWebSocketConnected } from '../../reducers';
+
+const initialState = {
+  dashboardInfo: [],
+  isLoaded: false,
+  isNotFinished: true,
+  error: ''
+};
+
+export /* istanbul ignore next */ class About extends Component {
+  constructor(props) {
+    super(props);
+    this.state = initialState;
+  }
+
+  componentDidMount() {
+    this.fetchDashboardInfo();
+  }
+
+  setErrorState(errorsFound) {
+    let errorMessage = 'Error getting version of ';
+
+    for (let i = 0; i < errorsFound.length; i += 1) {
+      if (i === 0) {
+        errorMessage += errorsFound[i];
+      } else if (i === errorsFound.length - 1) {
+        errorMessage += ` and ${errorsFound[i]}`;
+      } else {
+        errorMessage += `, ${errorsFound[i]}`;
+      }
+    }
+
+    if (this.state.isNotFinished) {
+      this.setState({
+        error: errorMessage,
+        isNotFinished: false
+      });
+    }
+  }
+
+  async fetchDashboardInfo() {
+    const dash = await getInstallProperties();
+    this.setState({
+      dashboardInfo: dash,
+      isLoaded: true
+    });
+  }
+
+  render() {
+    const { intl, loading } = this.props;
+    if (this.state.dashboardInfo !== null && this.state.isLoaded === true) {
+      const initialHeaders = [
+        {
+          key: 'property',
+          header: intl.formatMessage({
+            id: 'dashboard.tableHeader.Property',
+            defaultMessage: 'Property'
+          })
+        },
+        {
+          key: 'value',
+          header: intl.formatMessage({
+            id: 'dashboard.tableHeader.Value',
+            defaultMessage: 'Value'
+          })
+        }
+      ];
+
+      const initialRows = [];
+      let data = '';
+      const errorsToDisplay = [];
+
+      if (
+        this.state.dashboardInfo.InstallNamespace !== '' &&
+        this.state.dashboardInfo.InstallNamespace !== 'undefined'
```

Comment:
What's the scenario where the properties API would return a string 'undefined'? Seems unexpected. Can we change this value so it's easier to consume? `null` or an empty string would both work.

side: 'undefined' is a valid name for a namespace

Reply:
Returning an empty string / null from the cluster.go causes an error in the API in the checkstatus which causes non of the fields to render
Had to change it to return a non empty string to stop the unresolved promise error due to check status

Reply:
That's definitely unexpected, are you talking about this `checkStatus`? https://github.com/tektoncd/dashboard/blob/134bc627651c9e2d5fd139ae9e3933440e260962/src/api/comms.js#L35

It should only care about the status code, and if the content type is JSON that it's well-formed

Reply:
I have apparently done a fix to stop that from happening since I last saw it. Cant get it to happen again so it may have been a bad reaction to a return from somewhere else  
Will take out undefined and replace it with empty strings